package.json

```diff
@@ -50,7 +50,8 @@
     "eslint": "^3.15.0",
     "eslint-config-airbnb-base": "^11.1.0",
     "eslint-plugin-import": "^2.2.0",
-    "jest": "^20.0.4"
+    "jest": "^20.0.4",
+    "jest-environment-node-debug": "^2.0.0"
```

Comment:
I'm ok with this package. But then I think we should update readme and explain how to work with this + we should add an information about the workflow with ASTExplorer. That would make life of potential contributors easier. That being said, I'd prefer to move this to another PR and update Readme there as well. What do you think @jbandura ?